### PR TITLE
Autofill namespaces from kube config

### LINKED ISSUE #9: Options `--namespace` and `--sender-namespace` should be auto-filled with context namespace
Both flags `--namespace` and `--sender-namespace` (for send sub-command) are intended to be optional. If not provided they should be filled in from currently set namespace from a given _kubeconfig_ context.

I think the most logical place to do it is in following location - while creating `AddressableSpec` struct:

https://github.com/knative-sandbox/kn-plugin-event/blob/dd15346ae76b9e0c7168d2166ce45c491526a7c4/internal/cli/target.go#L71-L76

### PR DESCRIPTION
**Pull request by cardil**
# Changes

- :gift: Autofill of namespace and sender namespace from K8s config

/kind enhancement

Fixes #9 

**Release Note**

```release-note
From now on `--namespace` and `--sender-namespace` params for send command are being autoatically filled in with default values of KUBECONFIG.
```

### CODE REVIEW

File: pkg/configuration/memoized.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package configuration
```
Comment (dsimansk) [inline pkg/configuration/memoized.go:1]: The filename seems to have typo `memoized.go` or `memorized.go`?

File: pkg/configuration/momoized_test.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
+package configuration_test
```
Comment (dsimansk) [inline pkg/configuration/momoized_test.go:1]: Same question to filename as well.

File: pkg/configuration/memoized.go (hunk 1 above)
Comment (dsimansk) [reply pkg/configuration/memoized.go:1]: I see that's probably fine, but `*_test.go` has definitely a typo then. :)

Comment (cardil) [reply pkg/configuration/memoized.go:1]: This is fine. After: https://en.wikipedia.org/wiki/Memoization

Test file got typo.

File: pkg/configuration/momoized_test.go (hunk 2 above)
Comment (cardil) [reply pkg/configuration/momoized_test.go]: Fixed with f52ccea

**Review by dsimansk — APPROVED** [review]

**Pull request merged**